core/src/main/java/org/apache/iceberg/rest/responses/ErrorResponseParser.java

```diff
@@ -91,7 +91,7 @@ public static ErrorResponse fromJson(JsonNode jsonNode) {
     JsonNode error = jsonNode.get(ERROR);
     String message = JsonUtil.getStringOrNull(MESSAGE, error);
     String type = JsonUtil.getStringOrNull(TYPE, error);
-    Integer code = JsonUtil.getIntOrNull(CODE, error);
+    int code = JsonUtil.getInt(CODE, error);
```

Comment:
Question: if `CODE` isn't present, what exception will this throw?

Reply:
It would throw an IllegalArgumentException, the UT added with this PR shows the same.

Reply:
Oh my bad. The unit test completely shows that. I've been somewhat sick the past few days so forgive me 😅

Reply:
Not at all a problem :) !!